docs/source/gbd2017_models/causes/smoking_affected_causes/index.rst

```diff
@@ -0,0 +1,459 @@
+.. _2017_smoking_affected_causes:
+
+===============================================
+Smoking Affected Causes (Excluding Lung Cancer)
+===============================================
+
+The purpose of this cause model document is to represent the increased mortality risk associated with smoking exposure by combining the causes affected by the smoking risk factor. This specific cause model document was created to exclude lung cancer so that it can be used in a model that models the lung cancer cause directly.
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - SACs
+    - Smoking-affected causes
+    - 
+  * - ACMR
+    - All-cause mortality rate
+    - 
+  * - CSMR
+    - Cause-specific mortality rate
+    - 
+  * - EMR
+    - Excess mortality rate
+    - 
+  * - RF
+    - Risk factor
+    - 
+
+Disease Overview
+----------------
+
+Mortality among smokers is much higher than mortality among non-smokers. Therefore, it can be important to consider increased mortality attributable to smoking when smoking is included in a model, even if specific causes affected by smoking are not necessary to explicitly include in the model. See the causal framework section in the :ref:`Vivarium CSU Lung Cancer Screening Concept Model Documentation <2017_concept_model_vivarium_swissre_lungcancer>` for an example of when this is important.
+
```

Comment:
I wonder if we can mention the relative risk here too because that's what is really conferring the increased mortality risk associated with the smoking exposure- we apply the (1-PAF)*RR to the CSMRsacs.